rpmlint/checks/SpecCheck.py

```diff
@@ -83,6 +83,7 @@ def re_tag_compile(tag):
 tarball_regex = re.compile(r'\.(?:t(?:ar|[glx]z|bz2?)|zip)\b', re.IGNORECASE)
 
 python_setup_test_regex = re.compile(r'^[^#]*(setup.py test)')
+python_setup_install_regex = re.compile(r'^[^#]*(setup.py install|%\{?py(thon)?\d*_install)')
```

Comment:
`%python3.12_install` is not possible? That’s probably a request for comments from other distros.

Reply:
Not possible. Macro names don't have dots.

```
$ LANG=C rpm --define 'python3.12_install ha'
warning: Macro %python3 needs whitespace before body
warning: Macro %python3 needs whitespace before body
```

Reply:
Perfect, thank you.